serviceApp/src/main/java/domain/users/UserDTO.java

```diff
@@ -0,0 +1,121 @@
+package domain.users;
+
+import domain.activities.ActivityLevel;
+import domain.activities.SportDisciplines;
+import domain.location.Address;
+import validate.BornYear;
+import validate.EmailExist;
+import validate.Password;
+
+import javax.validation.constraints.*;
+
+
+@Password
+public class UserDTO {
+    @NotBlank(message = "nickname can't be empty")
+    @Size(min = 1, max = 16, message = "nickname should have 1 tp 16 characters long")
```

Comment:
tutaj chyba mała literówka "tp" -> "to". Ogólnie można później dograć z Julią ale wydaje mi się, że powinno być mniej więcej tak: "Your nickname must be between 1 and 16 characters".

Zastanawiam się czy nickname nie powinien mieć dolnej granicy wyższej 4-8 znaków i nie był liczbą. Teraz jest możliwość dodanie nicknam'u "2"